### PR TITLE
Lookup retention

### PR DESCRIPTION
**Pull request by lchapman4**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Review by bradleycamacho — APPROVED** [review]

File: src/content/docs/logs/ui-data/lookup-tables-ui.mdx [hunk 1]
```diff
@@ -52,6 +52,7 @@ Some more details about lookup tables:
 * Tables can only be queried from the [account](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#organization-accounts) in which the table was uploaded. 
 * Each [account](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#organization-accounts) has a limit of 20 lookup tables.
 * Lookup table data can't be used in NRQL alert conditions. 
+* Lookup tables have indefinite data retention by default.
```
Comment (zuluecho9) [inline src/content/docs/logs/ui-data/lookup-tables-ui.mdx:55]: what does 'indefinite' mean? this struck me as a bit unclear. if it exists forever, could we say that?

Comment (lchapman4) [reply src/content/docs/logs/ui-data/lookup-tables-ui.mdx:55]: Indefinite meaning we retain the data in lookup tables forever. We could absolutely rephrase it to say we retain lookup tables forever if that's clearer @zuluecho9.

Comment (lchapman4) [reply src/content/docs/logs/ui-data/lookup-tables-ui.mdx:55]: Hey @zuluecho9 is there anything I should change with the wording above? Should we not use indefinite to describe the data retention?

**Review by bradleycamacho — APPROVED** [review]
